src/test/regress/expected/multi_prepare_plsql.out

```diff
@@ -1233,9 +1233,31 @@ SELECT copy_in_plpgsql();
  
 (1 row)
 
+-- test prepared COPY on a non-distributed table
+CREATE TABLE local_ddl (x int);
+CREATE OR REPLACE FUNCTION local_copy_in_plpgsql()
+RETURNS VOID  AS
+$BODY$
+BEGIN
+    COPY local_ddl (x) FROM PROGRAM 'echo 1' WITH CSV;
+END;
+$BODY$ LANGUAGE plpgsql;
+SELECT local_copy_in_plpgsql();
+ local_copy_in_plpgsql 
+-----------------------
+ 
+(1 row)
+
+SELECT local_copy_in_plpgsql();
+ local_copy_in_plpgsql 
+-----------------------
+ 
+(1 row)
+
```

Comment:
I've added it but it can't really show correctness, just that we don't crash.

`COPY […] TO STDOUT` isn't permitted in PL/pgSQL, and writing to a file isn't something we do in tests, so that leaves sending it to a program, which works but doesn't really print any output.

I considered writing out to some sort of `csv` file in the results directory and then adding a correct `csv` file in the expected directory (they'd probably have to end in `out`) and adding a blank test with that name to trick `pg_regress`/`diff` into comparing the output and expectation of the `COPY TO` for us, but that seemed way too hacky.

So I've just added a test that verifies we don't crash.

Reply:
Hm, I've actually removed the test because I'm seeing weird behavior in Travis with COPY TO PROGRAM (namely a broken pipe). It no longer crashes, at any rate.